moderators/github.md

```diff
@@ -0,0 +1,55 @@
+# Maintaining Repositories
+
+This page is intended to provide guidelines for maintaining EddieHub's various repositories.
+
+## When To Merge PRs
+
+Pull requests should be merged only after:
+
+- At least 2 maintainers have provided an approving review.
```

Comment:
I would also suggest adding branch protection rules for the same 🤔

Reply:
That's something Eddie has to set up. We don't have access to the settings.

Additionally, there might be times where we want to allow single-review merges. 🤷🏻